x-pack/plugins/security_solution/public/flyout/left/components/analyze_graph.stories.tsx

```diff
@@ -32,9 +31,9 @@ export default {
 //   } as unknown as LeftPanelContext;
 //
 //   return (
-//     <LeftFlyoutContext.Provider value={contextValue}>
+//     <LeftPanelContext.Provider value={contextValue}>
 //       <AnalyzeGraph />
-//     </LeftFlyoutContext.Provider>
+//     </LeftPanelContext.Provider>
 //   );
```

Comment:
why keep these commented?:)

Reply:
good point, I've been thinking about deleting this file and the session view one for a while, so now is as good of a time as ever :)